stellarphot/utils/tests/test_magnitude_system_transforms.py

```diff
@@ -216,3 +233,21 @@ def test_refcats_adds_BVRI(self):
             ValueError, match="Transformer did not add columns for passbands"
         ):
             refcat.passband_columns(["X"], transformer=transform_refcat2_bands)
+
+
+class TestUSNOPrimeToSDSSDR7:
+    def test_transform(self):
+        # Check that the zero point for ug is correct
+        inp_mag = np.asarray([1.39, 0.0, 0.0, 0.0, 0.0, 1.0])
+        # This should produce this output:
+        # u: same as u' so 1.39
+        # g: product of -b_g and -zp_gr, so -0.0318
+        # r: product of -b_r and -zp_ri, so -0.00735
+        # i: product of -b_i and -zp_ri, so -0.00861
+        # z: product of -b_z and -zp_iz, so 0.0027
+
+        # This effectively checks the whole matrix
```

Comment:
Are there any test values we can use other than ones based on knowing the matrix based on observations (e.g. known ugriz to u'g'r'i'z' mapping)?